data_registry/admin.py

```diff
@@ -132,12 +134,35 @@ def queryset(self, request, queryset):
             return queryset.exclude(qs)
 
 
+class CustomDateFieldListFilter(DateFieldListFilter):
+    def __init__(self, *args, **kwargs):
+        super(CustomDateFieldListFilter, self).__init__(*args, **kwargs)
+
+        now = timezone.now()
+        # When time zone support is enabled, convert "now" to the user's time
+        # zone so Django's definition of "Today" matches what the user expects.
+        if timezone.is_aware(now):
+            now = timezone.localtime(now)
+
+        today = now.date()
+
+        self.links += ((
+            (_('More than a year ago'), {
```

Comment:
Since the last entry in the original filter is "This year", it might be a more logical order to have "Previous year" first and then "More than a year ago"